ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -302,6 +302,11 @@ extension BrowserViewController: WKNavigationDelegate {
 
     let isPrivateBrowsing = privateBrowsingManager.isPrivateBrowsing
     tab?.currentRequestURL = requestURL
+    if navigationAction.navigationType == WKNavigationType.backForward
+      || navigationAction.navigationType == WKNavigationType.reload
```

Comment:
nit: how about moving this to `isNewNavigation`, like `ads_tab_helper`

Reply:
Fixed